### DISCUSSION TITLE
How to use "Extensions" property for attachment field?

### DISCUSSION THREAD

**Discussion by peterhuppertz** [Budibase/budibase#8983] [Help] [answered]
Using the attachment field type I see a property "Extensions". I hoped to use that field to limit my uploads to PDF documents, so I tried pdf, PDF, *.pdf, etc. without success - the attachment field still accepted all file types. What is the correct way to use this property? Unfortunately, could not find hints in the docs or discussions.

**Comment by melohagan** [comment, marked as answer]
Hey @peterhuppertz 

This is a known bug! https://github.com/Budibase/budibase/issues/8724
Sorry for the inconvenience.
